src/pages/edit_resource/ResourceFormActions.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { Button, CircularProgress, Typography } from '@material-ui/core';
+import { publishResource } from '../../api/resourceApi';
+import { getStepLabel, ResourceFormStep, ResourceWrapper } from '../../types/resource.types';
+import { StyledContentWrapper, StyledSchemaPart } from '../../components/styled/Wrappers';
+import ErrorBanner from '../../components/ErrorBanner';
+import { useHistory } from 'react-router-dom';
+import { useFormikContext } from 'formik';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+
+const StyledButtonWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  max-width: ${({ theme }) => theme.breakpoints.values.md + 'px'};
+`;
+
+const StyledFormStatusWrapper = styled.div`
+  padding-right: 1rem;
+  display: inline-block;
+`;
+
+const StyledRightSideButtonWrapper = styled.div`
+  display: inline-block;
+`;
+
+const UpperCaseButton = styled(Button)`
+  text-transform: uppercase;
+`;
+
+const StyledArrowBackIcon = styled(ArrowBackIcon)`
+  margin-right: 1rem;
+`;
+const StyledArrowForwardIcon = styled(ArrowForwardIcon)`
+  margin-left: 1rem;
+`;
+
+interface ResourceFormActionProps {
+  activeStep: ResourceFormStep;
+  allChangesSaved: boolean;
+  setActiveStep: (step: ResourceFormStep) => void;
+}
+
+const ResourceFormAction: FC<ResourceFormActionProps> = ({ activeStep, allChangesSaved, setActiveStep }) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const [publishResourceError, setPublishResourceError] = useState(false);
+  const { values, dirty, isValid } = useFormikContext<ResourceWrapper>();
+
+  const handleNext = () => {
+    setActiveStep(activeStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep(activeStep - 1);
+  };
+
+  const handlePublishResource = async () => {
+    setPublishResourceError(false);
+    try {
+      await publishResource(values.resource.identifier);
+      history.push(`/resource/${values.resource.identifier}`);
+    } catch (error) {
+      setPublishResourceError(true);
+    }
+  };
+
+  return (
+    <StyledSchemaPart>
+      <StyledButtonWrapper>
+        {activeStep > 0 ? (
+          <UpperCaseButton variant="outlined" disabled={activeStep === 0} onClick={handleBack}>
+            <StyledArrowBackIcon /> {t(getStepLabel(activeStep - 1))}
+          </UpperCaseButton>
+        ) : (
+          <span />
```

Comment:
Etter mye prøving og feiling med flex-layout endte jeg opp med å legge inn en tomt element for å få layouten riktig når det mangler venstreknapp.